src/select/select.js

```diff
@@ -350,6 +359,40 @@ class Select extends React.Component<PropsT, StatelessStateT> {
     const {type, multiple} = this.props;
     return type === TYPE.search ? true : multiple;
   }
+
+  handledHotKeys(
+    // $FlowFixMe
+    e: SyntheticEvent<EventTarget> | KeyboardEvent,
+    option?: ?OptionT,
+  ) {
+    switch (e.key) {
+      case KEY_STRINGS.ArrowDown:
+      case KEY_STRINGS.Space:
+        if (e.key === KEY_STRINGS.Space && this.props.type === TYPE.search) {
```

Comment:
the `e.key === KEY_STRINGS.Space` seems redundant here since we're already in the case statement?

Reply:
oh wow, didn't notice the `case` fall-through, you're right. my bad.